components/HttpClient.tsx

```diff
@@ -0,0 +1,72 @@
+import { Alert } from "react-native";
+import ApiDictionary from '../constants/ApiDictionary';
+
+const state={
+    jwt:"",
+}
+
+export default async function bodyless(details: { destination: string; type: string;}) {
+
+    const response = await fetch(ApiDictionary.apiServer + details.destination , {
+        method: details.type,
+        headers: {
+            'Content-Type': 'application/json',
+            'jwt': state.jwt,
+        }})
+        .then(response => {
+          return response.json();})
+        .then(responseData => {
+            return responseData;})
+        .catch(err => {
+            console.log("fetch error" + err);
+            alert();
+        });
+        const resData = await response.json();
+        
+        if(response.ok) {
+            state.jwt = resData.token
```

Comment:
Hoezo wordt er altijd de JWT gezet bij een response 🤔

Reply:
dan hoef ik niet te differentiëren tussen inlog requests en andere requests, kan er indd nog een check in zetten dat het alleen inlog requests doet